### ISSUE TITLE
PBS TypeError crash

### ISSUE THREAD

**Issue by gjcooper** [nipy/nipype#1959]
### Summary
With a script that was previously running successfully with Multiproc I am now attempting to run on our Uni grid running PBS. The first couple of jobs submitted crash and return a TypeError.

Issue first occurred on release 0.12.1, tried upgrading to 0.13.0rc1 as it seemed to have new code realting to python3.
### Actual behavior
Trace for the crashes appears as:

```bash
170420-12:05:51,31 workflow ERROR:
         ['Node selectfiles.a1 failed to run on host rcglogin.']
170420-12:05:51,31 workflow INFO:
         Saving crash info to /home/gjc216/crash-20170420-120551-gjc216-selectfiles.a1-f1cc94d5-dfe1-41ae-b203-a8d7385aa7f9.pklz
170420-12:05:51,31 workflow INFO:
         Traceback (most recent call last):
  File "/home/gjc216/.virtualenvs/nipype/lib/python3.5/site-packages/nipype/pipeline/plugins/base.py", line 245, in run
    result = self._get_result(taskid)
  File "/home/gjc216/.virtualenvs/nipype/lib/python3.5/site-packages/nipype/pipeline/plugins/base.py", line 518, in _get_result
    if self._is_pending(taskid):
  File "/home/gjc216/.virtualenvs/nipype/lib/python3.5/site-packages/nipype/pipeline/plugins/pbs.py", line 55, in _is_pending
    return errmsg not in e
TypeError: a bytes-like object is required, not 'str'
```
### Expected behavior
Selectfiles should run successfully
### How to replicate the behavior
Pretty sure it is related to the unicode/str change in python3, so I am **assuming** that any python3 run ising PBS will fail
### Platform details:
```
{'scipy_version': '0.19.0', 'commit_source': 'installation', 'sys_version': '3.5.3 (default, Mar 17 2017, 08:49:30) \n[GCC 4.8.5 20150623 (Red Hat 4.8.5-11)]', 'traits_version': '4.6.0', 'networkx_version': '1.11', 'numpy_version': '1.12.1', 'nibabel_version': '2.1.0', 'sys_platform': 'linux', 'pkg_path': '/home/gjc216/.virtualenvs/nipype/lib/python3.5/site-packages/nipype', 'sys_executable': '/home/gjc216/.virtualenvs/nipype/bin/python', 'commit_hash': '4f08c9a'}
0.13.0-rc1
```

If you want me to try and attempt a pull-request I would be OK with that, although I would not have a python2 environment I could easily test on.

**Comment by mgxd** [comment]
@gjcooper thanks for reporting this - why don't you give it a try, our testing covers python 2-3, and we can help out if needed

**Comment by mgxd** [comment]
closed via #1987
